compiler/resolution/functionResolution.cpp

```diff
@@ -8687,7 +8687,7 @@ buildVirtualMaps() {
       continue;
 
     if (fn->hasFlag(FLAG_NO_PARENS))
-      // Parenthesesless functions are statically bound; that is, they are not
+      // Parentheses-less functions are statically bound; that is, they are not
```

Comment:
Although trivial, I think it'd be cleaner to keep spelling corrections out of this PR.